### PR TITLE
Sets are not serializable

### PR DESCRIPTION
**Pull request by jsf9k**
Sets are not serializable - at least not in BSON - so revert to using a list.

### CODE REVIEW

File: cyhy/mailer/cli.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def get_all_descendants(db, parent):
                 descendants.append(child)
                 descendants += get_all_descendants(db, child)
 
-    return set(descendants)
+    return descendants
```
Comment (dav3r) [inline cyhy/mailer/cli.py]: Why not do `list(set(descendants))` here?  That will remove duplicate org ids and still leave us with a list that can be safely passed into the mongo query.

**Review by dav3r — APPROVED** [review]
This is fine, but please see my comment.

File: cyhy/mailer/cli.py (hunk 1 above)
Comment (jsf9k) [reply cyhy/mailer/cli.py]: Great idea!  Change made in fe19393.

**Review by KyleEvers — APPROVED** [review]
👍

**Review by felddy — APPROVED** [review]
Set approves, and agrees he is not serializable.  He'd prefer Ra, but List is ok too.  
image

**Review by mcdonnnj — APPROVED** [review]
LGTM ✅

**Pull request merged**